bigbluebutton-html5/imports/ui/components/user-list/user-list-content/user-messages/component.jsx

```diff
@@ -120,8 +120,8 @@ class UserMessages extends Component {
           className={styles.scrollableList}
           ref={(ref) => { this._msgsList = ref; }}
         >
-          <div ref={(ref) => { this._msgItems = ref; }} className={styles.list}>
-            <TransitionGroup>
+          <TransitionGroup>
+            <div ref={(ref) => { this._msgItems = ref; }} className={styles.list}>
```

Comment:
This most likely breaks the animation.
TransitionGroup expects an array of CSSTransition objects to animate, but you wrapped that into one <div>